src/style.css

```diff
@@ -1,3 +1,112 @@
 body {
   margin: 0;
+  font-family: "Roboto", "Helvetica", sans-serif;
```

Comment:
it's better to use `scss` file extension instead of css since you will work much more with this preprocessor